src/module_wrap.cc

```diff
@@ -325,12 +325,12 @@ inline const struct read_result read_file(uv_file file) {
 }
 struct file_check {
   bool failed = true;
-  uv_file file;
+  uv_file file = -1;
 } file_check;
 inline const struct file_check check_file(URL search,
                                           bool close = false,
                                           bool allow_dir = false) {
-  struct file_check ret;
+  struct file_check ret = { true, -1 };
```

Comment:
Is this necessary if all fields have a default initializer?

Reply:
removed